pallets/schemas/src/types.rs

```diff
@@ -0,0 +1,23 @@
+use codec::{Decode, Encode};
+use common_primitives::schema::{SchemaId, SchemaResponse};
+use frame_support::{traits::Get, BoundedVec};
+use scale_info::TypeInfo;
+use sp_std::prelude::*;
+
+#[derive(Default, Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq)]
+#[scale_info(skip_type_params(MaxDataSize))]
+pub struct Schema<MaxDataSize>
```

Comment:
since we are going to store other metadata like `expired_block` related to `Schema` I think it makes sense to use a struct.

Reply:
I mean this would probably simplify the code that you added. But my real concern is that we do not do this. After todays discussion it looks like we are not clear on the direction we are going to take with this. Lets keep it simple for now.

Reply:
Converting back to `BoundedVec` for the sake of simplicity for now